solr/core/src/java/org/apache/solr/cloud/api/collections/ReindexCollectionCmd.java

```diff
@@ -711,7 +712,7 @@ private String getDaemonUrl(SolrResponse rsp, DocCollection coll) {
     // build a baseUrl of the replica
     for (Replica r : coll.getReplicas()) {
       if (replicaName.equals(r.getCoreName())) {
-        return r.getBaseUrl() + "/" + r.getCoreName();
```

Comment:
i love that this is simpler!  Lets crazy string parsing to create something and then later split the same thing!